index.js

```diff
@@ -25,5 +27,5 @@ module.exports = function(val) {
 }
 
 const _isTen = (val) => {
-  return val === 10;
+  return jQuery.equals(val, 10);
```

Comment:
```suggestion
  return jQuery.equals(jQuery.divide(val, jQuery.ten), jQuery.one);
```

https://github.com/cbrandolino/jQuery-basic-arithmetic-plugin/blob/master/jquery.basic_arithmetics.js#L94